content/manuals/engine/network/packet-filtering-firewalls.md

```diff
@@ -129,31 +129,69 @@ clients. No routes are normally set up in the host's network for container
 addresses that exist within a host.
 
 But, particularly with IPv6 you may prefer to avoid using NAT and instead
-arrange for external routing to container addresses.
+arrange for external routing to container addresses ("direct routing").
 
 To access containers on a bridge network from outside the Docker host,
 you must set up routing to the bridge network via an address on the Docker
 host. This can be achieved using static routes, Border Gateway Protocol
 (BGP), or any other means appropriate for your network.
 
-The bridge network driver has options
-`com.docker.network.bridge.gateway_mode_ipv6=<nat|routed>` and
-`com.docker.network.bridge.gateway_mode_ipv4=<nat|routed>`.
+Within a local layer-2 network, any remote host can set up a static route
+to a container network via the docker host's address on the local network.
+So, those local remote hosts can access containers directly. Remote hosts
+outside the local network will only have direct access to containers if
+routers are configured to enable it. 
```

Comment:
```suggestion
This allows local remote hosts to access containers directly. For remote
hosts outside the local network, direct access to containers requires
router configuration to enable the necessary routing.
```

Reply:
Hmm, reading this again, "local remote hosts" are a bit weird!

How about I change it to ...

```
Within a local layer 2 network, remote hosts can set up static routes
to a container network using the Docker daemon host's address on the local
network. Those hosts can access containers directly. For remote
hosts outside the local network, direct access to containers requires
router configuration to enable the necessary routing.
```

Reply:
Sounds good to me!